ansible/roles/uptime-kuma/templates/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.3'
+services:
+    uptime-kuma:
+        restart: always
+        ports:
+            - '127.0.0.1:3001:3001'
```

Comment:
There's not need to use `ports` here, only NGINX should be binding ports to the host. Instead use `expose`, which only exposes the port to the Docker network.

```
expose:
  - 3001
```